oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/authentication/JwtUserDetailsCheckingAuthenticationConverter.java

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
Please remove this from the PR as it is something that should be considered separately.